packager/file/file.cc

```diff
@@ -100,11 +101,11 @@ File* CreateUdpFile(const char* file_name, const char* mode) {
 }
 
 File* CreateHttpsFile(const char* file_name, const char* mode) {
-  return new HttpFile(HttpMethod::kPut, std::string("https://") + file_name);
+  return new HttpFile(HttpMethod::kPost, std::string("https://") + file_name);
```

Comment:
Why does this change from PUT to POST?  When I've tested this (briefly) with hard-coded auth tokens (that expire too quickly to be useful in production), the cloud services I wrote to all worked with PUT.  Do cloud storage services usually accept both POST and PUT?  Is POST specific to an auth proxy?

Reply:
The switch to POST was an intermediate step in the solution development process. The current LL-DASH actually implementation works with both PUT and POST, since we are just opening a file, writing to it, and then closing it. I just switched this back to `kPut` :) .